pyproject.toml

```diff
@@ -13,7 +13,7 @@ requires = [
     "meson-python>=0.5.0",
     "Cython>=0.29.21",
     "pybind11>=2.4.3",
-    "pythran>=0.9.12",
+    "pythran>=0.11.0",
```

Comment:
pretty sure this will create a merge conflict, but that's all being dealt with in gh-16353 anyway, where the caps are added.